db/knex_migrations/2023-09-13-1047-locale-on-statuspage.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function (knex) {
+    return knex.schema
+        .alterTable("status_page", function (table) {
+            table.boolean("show_locale_selector").notNullable().defaultTo(false);
+            table.string("default_locale").nullable().defaultTo("");
```

Comment:
being nullable and defaulting to a blang string is kind of strange db-design.

I think you meant either
```suggestion
            table.string("default_locale").nullable().defaultTo(undefined);
```
```suggestion
            table.string("default_locale").notNullable().defaultTo("");
```